.github/ISSUE_TEMPLATE/0-bug-report.yml

```diff
@@ -3,6 +3,19 @@ description: Report an issue with the Salesforce Code Analyzer Extension
 title: "[BUG] <YOUR_TITLE_HERE>"
 labels: []
 body:
+- type: dropdown
+  attributes:
+    label: Have you tried to verified this is Salesforce Code Analyzer Extension specific?
```

Comment:
Should be "tried to verify".

Reply:
Thanks... going with "Have you verified ..." instead.